packages/terra-application-utility/src/utility/_UtilityMenu.jsx

```diff
@@ -41,12 +41,20 @@ const propTypes = {
    * This will be provided by the terra-application-header or terra-application-menu.
    */
   onRequestClose: PropTypes.func,
+  /**
+   * Role attribute of Utility Menu.
+   */
+  role: PropTypes.string,
```

Comment:
in my opinion, we should name this `menuRole`, or something more specific.

Reply:
I would like to keep this as role since it would be more appropriate as we are referring to Role Attribute of element property name will be more suitable and it will be common across the other components as well. Let me know if there are any other reasons for which we need to change the prop name to menuRole from role.

Reply:
Role has been added to Application Menu utility and Application Header Utility and Application Menu Utility and Utility Button.

Reply:
My issue is that it is not clear which component we can set `role` for - a consumer needs to read the description to know they are setting `role` for UtilityMenu. Utility is tricky because it contains many sub components. Does `role` mean a consumer can set the role for the UtilityButton/UtilityMenu/UtilityDivider?

Reply:
hmm I missed that, yes it makes sense to keep prop name as menu role as it will be more specific to utility menu role. Changed prop name from role to menuRole.